docs/src/DeveloperDocumentation/changelog.md

```diff
@@ -0,0 +1,114 @@
+# Updating `CHANGELOG.md`
+
+Before every release, `CHANGELOG.md` should be updated. Most of this is ideally taken care
```

Comment:
```suggestion
Before every release, `CHANGELOG.md` should be updated. Most of this is ideally taken care of
```

Reply:
I'd say "ideally" should also be moved to the beginning of the sentence.

Reply:
```suggestion
Before every release, `CHANGELOG.md` should be updated. Ideally most of this is taken care of
```